tests/myria3d/models/test_model.py

```diff
@@ -0,0 +1,28 @@
+import hydra
+from pytorch_lightning import LightningDataModule
+
+from myria3d.models.model import Model
+
+
+def test_model_get_batch_tensor_by_enumeration():
+    with hydra.initialize(config_path="./../../../trained_model_assets/", job_name="config"):
+        config = hydra.compose(
+            config_name="proto151_V2.0_epoch_100_Myria3DV3.1.0_predict_config_V3.7.0",
```

Comment:
This config_name is hardcoded. Can it be defined imported (I think it is defined in run.py or train.py).

Reply:
Thanks, looks a good idea, but I'm not sure it is easy to get DEFAULT_CONFIG_FILE from `run.py` as it is not in the myria3d python module (hence I suppose it is not intended to be imported)

Reply:
Inded, but consequently I do not think it is supposed to be used for testing. 
In test I usually used function `make_default_hydra_cfg` from `conftest.py`, to always test for the default configuration.

Reply:
I finally managed to find how to use it (the issue was that datamodule was using hydra:runtime.cwd, so I had to override the parameter that required it